static/css/components/page-banner.less

```diff
@@ -62,5 +62,6 @@
     padding: 10px;
     line-height: 25px;
     text-align: center;
+    display: block;
```

Comment:
^^ Further up in this file, there's `display:none` on the main `.page-banner` element; let's knock that off, and the corresponding media query that displays the banner on small screens.